README.md

```diff
@@ -59,6 +59,40 @@ spec:
 
 This will create a new Sandbox named `my-sandbox` running the image you specify. You can then access the Sandbox using its stable hostname, `my-sandbox`.
 
+For a more programmatic approach you can use the [SDK](clients/python/agentic-sandbox-client/README.md) like the following:
+```python
+from agentic_sandbox import Sandbox
```

Comment:
This looks incorrect. It should be `SandboxClient`. 

Instead of embedding the full example here, the link to the Python client README already contains detailed usage instructions. If we have to keep an example here, keep it minimal. 

Given the Python client could change over time, keeping the example closer to its code seems better, and avoid issues like this one.